pages/sdk/relay-kit/guides/migrate-to-v4.md

```diff
@@ -0,0 +1,63 @@
+# Migrate to v4
+
+This guide references the major changes between v3 and v4 to help those migrating an existing app.
+
+## Default `safeModuleVersion` is 0.3.0
```

Comment:
Maybe warning that api-kit currently is only compatible with safeModuleVersion v0.2.0?